### PR TITLE
SPOI-8652 Example for Enricher

### PR DESCRIPTION
**Pull request by chinmaykolhatkar**
Example for Enricher.

@sandeepdeshmukh Can you please review and merge?

### CODE REVIEW

**Comment by chinmaykolhatkar** [comment]
@bhupeshchawda @sandeepdeshmukh Can you please review and merge?

File: tutorials/README.md [hunk 1]
```diff
@@ -14,3 +14,4 @@
 | partition         | Shows use of custom partitioning with a stream codec.
 | topnwords         | The Top-N words example. |
 | jdbcToJdbc        | This application reads from an input table using JDBC, converts input to user defined POJO & then writes those POJOS to another table. |
+| enricher          | Shows how to enrich streaming data using external source. |
```
Comment (bhupeshchawda) [inline tutorials/README.md]: Alphabetic ordering

File: tutorials/enricher/src/site/conf/my-app-conf1.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (bhupeshchawda) [inline tutorials/enricher/src/site/conf/my-app-conf1.xml]: Remove this file if not being used. Might create unnecessary confusion.

Comment (chinmaykolhatkar) [inline tutorials/enricher/src/site/conf/my-app-conf1.xml]: Its part of standard template of apex app archetype. Hence kept it as it is. All other examples have also done the same.

File: tutorials/README.md (hunk 1 above)
Comment (chinmaykolhatkar) [inline tutorials/README.md]: Done.

**Pull request merged**